README.md

```diff
@@ -219,14 +219,99 @@ This script uses [dtslint](https://github.com/microsoft/DefinitelyTyped-tools/tr
 
 Once you have all your changes ready, use `pnpm run test-all` to see how your changes affect other modules.
 
+##### @arethetypeswrong/cli (`attw`) checks
+
+dtslint includes module format and `package.json` configuration checks from [@arethetypeswrong/cli](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/packages/cli). The checks run only if a SemVer-major-compatible implementation package can be found on npm to compare against the DefinitelyTyped package. (DefinitelyTyped packages marked as `"nonNpm": true` in their `package.json` are skipped.)
+
+Many packages currently fail the `attw` checks and need to be fixed. To allow us to make incremental progress, failed `attw` checks do not fail the `dtslint` run when the package is listed in `failingPackages` in [`attw.json`](./attw.json), but they will still be reported in the `pnpm test my-package` output. Once the problems have been fixed, the package must be removed from `failingPackages`, and `attw` compliance will be enforced going forward.
```

Comment:
```suggestion
Many packages currently fail the `attw` checks and need to be fixed. To allow us to make incremental progress, failed `attw` checks do not fail the `dtslint` run when the package is listed in `failingPackages` in [`attw.json`](./attw.json), but they will still be reported in the `pnpm test my-package` output. If you fix the package, remove it from `failingPackages` so that `attw` checks can start failing `dtslint` runs.
```